serial_driver/src/serial_port.cpp

```diff
@@ -73,6 +73,11 @@ void SerialPort::async_receive(Functor func)
     });
 }
 
+void SerialPort::send_break()
+{
+  m_serial_port.send_break();
```

Comment:
Can you please check for `is_open()` here before trying to send the break?

Reply:
Yep, want me to change the signature to `bool SerialPort::send_break()` then and return false if the port isn't open?

Reply:
Sure. The non-async versions of send/receive return meaningful values so we might as well be consistent.

Reply:
Got that added with some docs. Writing a unit test for it next.